admin/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copy template files into <build_dir>/horace_config
+set(WORKER_CONFIG_DIR "${CMAKE_BINARY_DIR}/horace_config")
+file(MAKE_DIRECTORY ${WORKER_CONFIG_DIR})
+
+set(_path_regex "[a-zA-Z0-9/\\\\_\\-\\.: ]+")
```

Comment:
the comment we have in the Herbert file explaining this regex needs to be copied here